src/components/PageNavigationButtons/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import './PageNavigationButtons.scss';
+
+
+const PageNavigationButtons = props => (
+  <div className="page-nav-btns">
+    {
+      props.prev.trim() ?
+        <Link className="btn btn-primary" to={props.prev}>Previous</Link>
```

Comment:
1) These are not paragon buttons (or buttons at all)
2) Since these are used to redirect, the accessible element for these is an `<a>` tag instead of a `<button>`. Also we need to use the `<Link>` component from react router for it work correctly with routing history. The className just styles the anchor tag like a button. 

I will swap out the disabled buttons with disabled Paragon buttons though.

Reply:
got it thanks!
LGTM